test/04_openst/01_pricer_rule.js

```diff
@@ -0,0 +1,58 @@
+// 'use strict';
+
+const assert = require('assert');
+const BN = require('bn.js');
+
+const { ContractInteract } = require('@openst/openst.js');
```

Comment:
external npm requires should be together as per eslint. Below 3 can go totgether. 
After that a new line can be added.
```
const assert = require('assert');
const BN = require('bn.js');
const { ContractInteract } = require('@openst/openst.js');
```